cartography/cli.py

```diff
@@ -587,7 +587,7 @@ def run_gcp(request):
         neo4j_max_connection_lifetime=request['neo4j']['connection_lifetime'],
         credentials=request['credentials'],
         params=request['params'],
-        gcp_requested_syncs=request['services'],
+        gcp_requested_syncs=request.get('services', None),
```

Comment:
if `services` is not passed, we should use the default services list. Otherwise, all services would be skipped.

Reply:
for this we already have logic in  `__init.py__`  file
`requested_syncs: List[str] = list(RESOURCE_FUNCTIONS.keys())
    if config.gcp_requested_syncs:
        requested_syncs = parse_and_validate_gcp_requested_syncs(config.gcp_requested_syncs)`